### DISCUSSION TITLE
Is version 2.6.0 is still latest release?

### DISCUSSION THREAD

**Discussion by toge** [edenhill/librdkafka#4910] [General]
This is a minor issue.
2.6.1 has already been released, but the Github release lists 2.6.0 as the latest release.
Is there a reason for this?

![image](https://github.com/user-attachments/assets/46da8a7b-95a7-42b5-b54a-68a5a6f5d5ae)

**Comment by marlowa** [comment]
I am not sure that librdkafa consistently supports semantic versioning. I think it doesn't. I am on version 2.2.0 which has a significant bug in the area of idempotency. That bug will not be fixed in 2.2.1. There probably won't be a 2.2.1. I (and my team on the project I am on) have been advised by confluent that when bugs like this are found they will be fixed on the latest version and their advice is for us to move to the latest version as and when such fixes are made in it. I complained that by calling versions 2.2.0 and 2.6.1 in your case, they are giving the illusion of supporting semantic versioning, which is very misleading. I asked them to make a public statement to make their versioning policy clear. I am still waiting for this.

**Comment by mensfeld** [comment]
@toge 
![obraz](https://github.com/user-attachments/assets/ef5a9d81-b3d9-4174-8997-b8a21c5189d1)

i think it's good now.

**Comment by toge** [comment]
Thanks a lot!
